CryptoPrices/Sources/Domain/Sources/DomainTestHelpers/MockCoinDetail/MockMarketData.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  MockMarketData.swift
+//
+//  Created by Khanh on 28/12/2022.
+//
+
+import Entities
+
+public struct MockMarketData: MarketData, Equatable {
+
+    public let currentPrice: USDDecimalable
+    public let priceChangePercentage24H: Double
+    public let marketCap: USDDecimalable
+    public let marketCapChangePercentage24H: Double
+    public let ath: USDDecimalable
+    public let athChangePercentage: USDDoublable
+    public let atl: USDDecimalable
+    public let atlChangePercentage: USDDoublable
+    
+    public static func == (lhs: MockMarketData, rhs: MockMarketData) -> Bool {
+        lhs.currentPrice.usd == rhs.currentPrice.usd
+    }
```

Comment:
Just curious, is it intentional that we ignore other attributes and only do the comparison for `currentPrice.usd`? Shouldn't we make the comparisons for other fields too? 🤔

Reply:
@minhnimble Thanks for the question.`MockMarketData` is an attribute of `MockCoinDetail`, at `MockCoinDetail` I check their `id` for comparison, and I think comparing `id` is "enough". Therefore, I intended to only compare `currentPrice.usd`. Please correct me if I was wrong 🙏 .

Reply:
Noted, if it is okay that we only need the id to be the same then it is good to me too. We can revisit this again when there are any other issues come up 🙏